src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/EventServiceImpl.java

```diff
@@ -190,12 +190,14 @@ public Event deleteEvent(UUID collexUuid, String tag) throws CTPException
      * @param messageType the type of change
      * @param event the event to which the change occurred
      */
-    private void fireEventChangeHandlers(final CollectionExerciseEventPublisher.MessageType messageType, final Event event) {
+    private void fireEventChangeHandlers(final CollectionExerciseEventPublisher.MessageType messageType,
+                                         final Event event) {
         Arrays.stream(this.changeHandlers).forEach(handler -> {
             try {
                 handler.handleEventLifecycle(messageType, event);
             } catch (CTPException e) {
-                log.error("Failed to handle event change for {} of {} - {}", messageType, event, e);
+                log.error("Failed to handle event change for {} of {} - {} ({})",
```

Comment:
What's the reason for surprising the stacktrace?

Reply:
Reduce noise in the logs - these can happen alot and they are mostly harmless